Pcap++/src/DpdkDevice.cpp

```diff
@@ -8,6 +8,11 @@
 #include "DpdkDevice.h"
 #include "DpdkDeviceList.h"
 #include "Logger.h"
+#include "rte_version.h"
+#if (RTE_VER_YEAR >= 16) && (RTE_VER_MONTH > 07)
+#include "rte_bus_pci.h"
+#endif
+#include "rte_pci.h"

```

Comment:
This is the line which causes DPDK 16.11 build to fail. If you can add this line one line up, so it's inside the `#if (RTE_VER_YEAR >= 16) && (RTE_VER_MONTH > 07)` it'd great